src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/BindingInfo.cs

```diff
@@ -113,6 +120,13 @@ public static BindingInfo GetBindingInfo(IEnumerable<object> attributes)
                 bindingInfo.PropertyFilterProvider = new CompositePropertyFilterProvider(propertyFilterProviders);
             }
 
+            // RequestPredicate
+            foreach (var requestPredicateProvider in attributes.OfType<IRequestPredicateProvider>())
+            {
+                isBindingInfoPresent = true;
+                bindingInfo.RequestPredicate = requestPredicateProvider.RequestPredicate;
```

Comment:
Should this `break` after the first item (similar to `BindingSource`)? It's picking predicate arbitrarily anyway